### DISCUSSION TITLE
Community (Server side) Swift Starter Project

### DISCUSSION THREAD

**Discussion by maartene** [BattlesnakeOfficial/feedback#256] [General]
Hello, 

I created a Swift starter project (based on the JavaScript one). Can it be added to the community list of starter projects?
It uses [Vapor](https://vapor.codes) as the server framework. 

You can find it here: https://github.com/maartene/BattlesnakeSwiftStarter

Kind regards,

Maarten

**Comment by bvanvugt** [comment]
@maartene Thanks for this! I've added it to the [starter project page](https://docs.battlesnake.com/starter-projects) :)
